src/Runtime/rigidBody.ts

```
@@ -99,6 +99,16 @@ class RigidBodyInner {
     public get hasShadows(): boolean {
         return 0 < this._shadowCount;
     }
+
+    public getShapeReference(): PhysicsShape {
+        return this._shapeReference!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)